resources/assets/components/CampaignInbox/index.js

```diff
@@ -61,4 +62,31 @@ class CampaignInbox extends React.Component {
   }
 }
 
+CampaignInbox.propTypes = {
+  campaign: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+  deletePost: PropTypes.func.isRequired,
+  displayHistoryModal: PropTypes.bool,
+  hideHistory: PropTypes.func.isRequired,
+  historyModalId: PropTypes.number,
+  loading: PropTypes.bool.isRequired,
+  postIds: PropTypes.array, // eslint-disable-line react/forbid-prop-types
```

Comment:
This could be `PropTypes.arrayOf(PropTypes.string)` to silence this linter warning.